pkgs/applications/audio/pyradio/default.nix

```diff
@@ -36,6 +39,9 @@ python3Packages.buildPythonApplication rec {
   meta = with lib; {
     homepage = "http://www.coderholic.com/pyradio/";
     description = "Curses based internet radio player";
+    longDescription = ''
+      Needs mpv, mplayer or vlc installed as a backend.
```

Comment:
That seems rather random. Do we have some doc we can link instead?

Reply:
> That seems rather random. Do we have some doc we can link instead?

I've removed that section.